maps-ktx/src/main/java/com/google/maps/android/ktx/MapsExperimentalFeature.kt

```diff
@@ -0,0 +1,7 @@
+package com.google.maps.android.ktx
```

Comment:
I believe this file can be removed? Or are there other annotated APIs remaining?

Reply:
I'd like to keep this in here for future experiments—could be useful

>Also, the demo app in /app still has the @MapsExperimentalFeature annotation - that could be removed as well. I believe the only reason that's still building is that the annotation class is still present.

I initially removed it but then reverted it back in a3d58a3 since the demo app references the maven central artifact. Will remove that once the version is bumped in the demo app.

Reply:
Makes sense!